app/controllers/points_controller.rb

```diff
@@ -19,12 +20,23 @@ def show; end
 
   # GET /points/new
   def new
+    @accounts = Account.all
+    @current_user = @accounts.find_by uuid: session[:useruuid]
     @point = Point.new
     @account = Account.find(params[:account_id])
+    unless @current_user.status_id == 2
```

Comment:
We can provide access to both admin and the officer (ID: 1, 2). Approving since that wasnt the scope of user story